lib/school.rb

```diff
@@ -0,0 +1,20 @@
+class School
+  def initialize(start_time, hours_in_school_day)
+  end
+
+  def start_time
+    @start_time = '9:00'
+  end
+
+  def hours_in_school_day
+    @hours_in_school_day = 7
+  end
```

Comment:
This will work for the exact school that is in our test, but is not very dynamic - not every school will start at 9:00 and be 7 hours long.  A better solution is to create these instance variables in the initialize method, using the information we send into our arguments:

```ruby
class School
  attr_reader :start_time, :hours_in_school_day, :student_names
  
  def initialize(start_time, hours_in_school_day)
    @start_time = start_time
    @hours_in_school_day = hours_in_school_day
    @student_names = []
  end
end